pkg/util/util.go

```diff
@@ -164,3 +166,32 @@ func CreateTempDir(path string) (string, error) {
 	dir, pattern := getTempPathArgs(path)
 	return os.MkdirTemp(dir, pattern)
 }
+
+func IsOSTree() bool {
+	exists, err := PathExists("/run/ostree-booted")
+	if err != nil {
+		return false
+	}
+	return exists
+}
+
+func IsBootC() bool {
```

Comment:
They're using `bootc`, not `BootC` or `bootC` (https://bootc-dev.github.io/bootc/), so I think this is more accurate 
```suggestion
func IsBootc() bool {
```